bindings/java/c/evmc-vm.c

```diff
@@ -24,7 +24,7 @@ JNIEXPORT jobject JNICALL Java_org_ethereum_evmc_EvmcVm_init(JNIEnv* jenv,
         {
             const char* error_msg = evmc_last_error_msg();
             jclass jclazz = (*jenv)->FindClass(jenv, "java/lang/AssertionError");
-            (*jenv)->ThrowNew(jenv, jclazz, error_msg);
+            (*jenv)->ThrowNew(jenv, jclazz, error_msg ? error_msg : "Loading EVMC VM failed");
```

Comment:
`ThrowNew` expects a valid pointer and `evmc_last_error_msg` can return null.

Reply:
Is there really a crash happening?

Reply:
https://docs.oracle.com/javase/8/docs/technotes/guides/jni/spec/functions.html

The documentation is not explicit about it, but says:
> message: the message used to construct the java.lang.Throwable object. The string is encoded in modified UTF-8.